doc-ros2-documentation.yaml

```diff
@@ -0,0 +1,19 @@
+%YAML 1.1
+# ROS buildfarm ROS 2 documentation build file
+---
+documentation_type: docker_build
+doc_repositories:
+  ros2_doc:
+    url: https://github.com/ros2/ros2_documentation.git
+    branch: rolling
```

Comment:
Checking my understanding out loud: With multisphinx the documentation for the other distributions will also be built even though the branch here is `rolling` correct? We won't need separate "independent" documentation job configurations for each distribution?

Reply:
Yes, exactly.  The way that `sphinx-multiversion` works is that you tell it which branches you want to build.  It then checks each branch of the git repository out in turn, builds the documentation, and combines it all together in one final output directory.  So as-is, this will build for all of the active branches.